tutorials/desktop/firefox-for-web-developers/firefox-for-web-developers.md

```diff
@@ -0,0 +1,309 @@
+
+![CSS rules for the <h2> element](images/rules_panel.png)
+
+The CSS rules include an entry that's setting the `margin-bottom` property to `2.25em`. Move the mouse over any rule and you'll see that checkboxes appear, letting you turn individual rules on and off wihout having to edit them. Turn off the `margin-bottom` rule to see what effect it's having on your heading. Turning it off completely makes the spacing a little too tight. Double-click on the value and change it to `1em` instead.
+
+Finally let's give our new element a bit of colour by adding a new rule. First we need to choose a colour to use and, as we want our changes to fit in with the rest of the page, it seems appropriate to use the same shade of orange that's in the page header. But what shade is that, exactly? Fortunately the inspector has an eyedropper tool that will let us pick the colour of any pixel in our web page. It's up at the top right of the DOM section, next to the search field. Just click the button and move your mouse back into the page area, where you'll discover the pointer has changed into a magnifying loupe that makes it easy to pick a specific pixel. Click in the orange header and the hexadecimal colour value will be copied to your clipboard.
+
+Now to apply that value to our heading. With the `<h2>` still selected in the DOM panel, move to the CSS rules section on the right. Towards the top is an empty rules section labelled "element". Click on the blank white space in that section and a text entry cursor will appear. To add a new CSS rule you simply have to enter the name of the rule (`color` in this case — note the American spelling), then press Enter or Tab to enter the value. In this case we're pasting the value from the clipboard, so just press Ctrl-V rather than typing something in.
+
+If everything went well your website now looks something like this:
+
+![The web page after editing in the inspector](images/edited_page.png)
+
+
+### Tidying a page for printing ###
+We're going to do one last thing in the inspector: delete something. Right-click on your new `<h2>` element in the DOM list and select **Delete Node** from the menu. The element is immediately removed from the page. Have you ever been annoyed when printing a web page only to find that your toner or ink is wasted on advertisements, logos or other parts that weren't necessary. Now you know how to remove unwanted content from the page before you print.
+
+There's lots more that you can do in the inspector. If you're learning to write HTML it can be invaluable to help understand how other pages are put together, and if you're trying to design a page then being able to see the effects of CSS changes immediately is a huge benefit. Play around and don't be afraid to break things — **F5** will soon get you back to the original page.
```

Comment:
Could you add links to HTML and CSS tutorials, in case the reader wants to learn those things? But please not w3schools. How about https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML and https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS ?

Reply:
Actually I'm just going to do this for you. Let me know if you object.

Reply:
The tutorial guidelines are strongly against including links in the body of the content that could send users off to another site. I do have a link on the last page of the tutorial to the main MDN start page. I'm not against adding more specific links to that list, though.

Reply:
Okay I've removed my commits to add links.